koku/api/report/ocp/serializers.py

```diff
@@ -129,6 +130,16 @@ class OCPQueryParamSerializer(ReportQueryParamSerializer):
     FILTER_SERIALIZER = OCPFilterSerializer
     EXCLUDE_SERIALIZER = OCPExcludeSerializer
 
+    def to_internal_value(self, data):
+        """Send to internal value."""
+        if delta_value := data.get("delta"):
+            if isinstance(delta_value, str):
```

Comment:
Is it possible for `delta_value` to be anything other than a string here?

Reply:
Yes, if you pass in more than two delta params:
```
?delta=distributed_cost&delta=cost
```
It will turn it into a list causing the param value to be `["distributed_cost", "cost"]`, Which causes an error when passed into the `.get()`. More than one delta is not supported anyway, so this will throw a validation error instead.